_health-care/_js/_form.js

```diff
@@ -1241,6 +1285,9 @@ var HealthApp = (function() {
           e.preventDefault();
           // TODO(awong): Remove this debugging logic when things work.
           console.log(build1010ezXml(formRoot));
+          for (var i = 0; i < allInputs.length; i++) {
+            validate(allInputs[i]);
```

Comment:
Ah you might be right, I will try moving it into the change handler once the "Continue" buttons are built!